### PR TITLE
Fetch additional version data for beta build id's

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

**Comment by wlach** [comment]
@maurodoglio this was failing before because of a few silly things but should be good for review now

File: tests/test_etl_measure.py [hunk 1]
```diff
@@ -64,3 +78,20 @@ def test_update_measure_with_initial_data(mock_raw_query, mock_raw_query_data,
             sorted([(d[0], d[3], d[4]) for d in mock_raw_query_data], key=lambda d: d[0])
         }
     }
+
+
+@freeze_time('2017-07-01 13:00')
+@responses.activate
```
Comment (maurodoglio) [inline tests/test_etl_measure.py]: I would a new [responses fixture](https://github.com/getsentry/pytest-responses/blob/b2e22923eb3814144733ce93f68b0d8506c0bf76/pytest_responses.py#L22-L25) and include it in the test dependencies when needed; that would save yousome typing and it feels more pytest-y 😺

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+def pytest_runtest_setup(item):
+    """
+    Per-test setup.
+    - Provide cache isolation incrementing the cache key prefix
+    """
+    increment_cache_key_prefix()
+
+
+def increment_cache_key_prefix():
```
Comment (maurodoglio) [inline tests/conftest.py]: Why don't we use cache.clear()? These tests will never run on production, so there's no risk of loosing data by accident.

File: requirements.txt [hunk 3]
```diff
@@ -273,3 +273,13 @@ vine==1.1.4 \
 freezegun==0.3.9 \
     --hash=sha256:8842688de9497c82ecb40c30274ecc676e97ac78765b0ade105a97063d5b7a11 \
     --hash=sha256:783ccccd7f60968bfe49ad9e114c18ea2b63831faaaf61c1f1f71ddfde1c0eee
+cookies==2.2.1 \
+    --hash=sha256:15bee753002dff684987b8df8c235288eb8d45f8191ae056254812dfd42c81d3
+responses==0.7.0 \
+    --hash=sha256:860f2ac3aed6954b6dfc79ee27a229d423c77ad7a4c242f26c1489bdd702d8e4
+pytest==3.2.1 \
```
Comment (wlach) [inline requirements.txt]: Oops, I must have missed that. I added these when I was trying to get the cache clearing working. I guess we don't need them, will take this out.

File: missioncontrol/etl/versions.py [hunk 4]
```diff
@@ -22,3 +27,34 @@ def get_firefox_versions():
...
+
+def get_version_string_from_buildid(channel, buildid):
+    '''
+    Utility function for getting a human-readable version string based
+    on a buildid and channel. Generally only needed for beta (where
+    the version doesn't specify the beta number)
+    '''
+    cache_key = _get_version_string_cache_key(channel, buildid)
+    cached_result = cache.get(cache_key)
+    if cached_result:
```
Comment (wlach) [inline missioncontrol/etl/versions.py]: I like that idiom, I'll use it.

File: tests/conftest.py (hunk 2 above)
Comment (wlach) [inline tests/conftest.py]: Yeah, I cribbed that code from treeherder. But you're right, just clearing the cache is the obvious solution. I have no idea why we used this complicated piece of code there --  looks like it was actually added by you? :)

https://github.com/mozilla/treeherder/commit/9fd2f645429b2883a1097f9ddb933a34de49e359

/cc @edmorley

**Comment by maurodoglio** [comment]
It was inherited from datazilla and the idea there was to avoid the risk of
clearing the cache on the prod system. I don't think we should worry about
that.

**Comment by wlach** [comment]
Merged with requested changes

**Pull request merged**